specification/security/resource-manager/Microsoft.Security/preview/2024-07-01-preview/trustedIps.json

```diff
@@ -0,0 +1,323 @@
+          }
+        },
+        "tags": [
+          "TrustedIps"
+        ],
+        "description": "Delete a trusted IPs object over a given scope",
+        "operationId": "TrustedIps_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/Scope"
+          },
+          {
+            "$ref": "#/parameters/TrustedIpsId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "204": {
+            "description": "No Content - Trusted IPs does not exist"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "TrustedIpsList": {
+      "type": "object",
+      "description": "Page of a trusted IPs list",
+      "properties": {
+        "value": {
+          "description": "Collection of trusted IPs in this page",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TrustedIps"
+          }
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The URI to fetch the next page"
+        }
+      }
+    },
+    "TrustedIps": {
+      "type": "object",
+      "description": "Trusted IPs over a given scope",
+      "properties": {
+        "properties": {
+          "description": "Properties of a trusted IPs object",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/TrustedIpsProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
+        }
+      ]
+    },
+    "TrustedIpsProperties": {
+      "type": "object",
+      "description": "Describes properties of a trusted IPs",
+      "properties": {
+        "tenantId": {
+          "description": "The tenantId (GUID)",
+          "readOnly": true,
+          "type": "string"
+        },
+        "displayName": {
+          "description": "Display name of the trusted IPs",
+          "type": "string"
+        },
+        "inheritedFrom": {
+          "description": "The scope from which the trusted IPs object was inherited; if not applicable, this will be null",
+          "readOnly": true,
+          "type": "string"
+        },
+        "cidr": {
+          "description": "Specifies the trusted IPs range in CIDR notation",
+          "type": "string"
+        },
+        "metadata": {
+          "$ref": "#/definitions/TrustedIpsMetadata"
+        }
+      },
+      "required": [
+        "displayName",
+        "cidr"
+      ]
+    },
+    "TrustedIpsMetadata": {
```

Comment:
This looks a lot like systemData from ARM - how does it differ? does it have both?

Reply:
No, this is not related to arm, this is the data (internet configuration) set up by the customer. 
I recommend to read our design doc to get more details.